### DISCUSSION TITLE
OpenRCT2 on the Play Store?

### DISCUSSION THREAD

**Discussion by BCullen2201** [OpenRCT2/OpenRCT2#17275] [Q&A] [answered]
Would be nice for people on Chromebook, they aren't allowed to install from .apk

**Comment by janisozaur** [comment, marked as answer]
it's not feasible until we address several issues, big and small, related to Android builds, such as sourcing the assets, embedding g2.dat, improving UI... but nobody is really actively working on that, because we usually find more interesting/pressing things to address. #17270 might help move that.
